_summaries/srivastava2014dropout_1.md

```diff
@@ -1,19 +1,114 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: srivastava2014dropout
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: patelojas
+score: 9
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+# Main Idea
+
+This paper looks to solve the problem of overfitting for large networks. It introduces dropout, a technique that randomly drops units and their connections from a neural network during training. 
+
+Current deep learning networks contain many layers, allowing them to learn complicated relationships from data. However, due to limitations in training data and the depth and complexity of modern data, some of the relationships are learned from overfitting and will not be seen in actual test data. 
+
+Ensemble methods work very well in many settings. These methods include combining several models, but is tough when considering large neural networks. Ensemble methods are the most effective when each individual model is unique from the rest, such as having a different architecture or being trained on different data. Given the current limits on computation power and smaller amounts of training data, having several unique networks is a difficult task. Instead, dropout simulates combining different neural networks together. 
```

Comment:
It is good to discuss the motivation of implicit ensemble behind dropout. But the current version seems a bit lengthy.